src/timetrackings/index.ts

```diff
@@ -40,4 +40,13 @@ export default class TimeTrackings extends BaseApi {
     const response = this._put<TimeTracking>(`${TimeTrackings.resourceName}/stop`, data);
     return responseHandler.required(response);
   }
+
+  public current(userId?: number): Promise<TimeTracking | undefined> {
+    let query = '';
+    if (userId) {
+      query = `?user_id=${userId}`;
+    }
+    const response = this._get<TimeTracking>(`${TimeTrackings.resourceName}/current${query}`);
```

Comment:
Pls. use the options AxiosRequestConfig to add query  

This should work  this._get<TimeTracking>(`${TimeTrackings.resourceName}/current, { params: {user_id: userId} } );